byron/chain/executable-spec/src/Cardano/Spec/Chain/STS/Rule/Chain.hs

```diff
@@ -56,12 +74,41 @@ instance STS CHAIN where
     | BBodyFailure (PredicateFailure BBODY)
     | PBFTFailure (PredicateFailure PBFT)
     | MaximumBlockSize Natural Natural
+    | LedgerUTXOFailure (PredicateFailure (UTXOWS TxId))
+    | LedgerDelegationFailure (PredicateFailure DELEG)
     deriving (Eq, Show)
 
-  initialRules = []
+  initialRules =
```

Comment:
How about putting this into a separate PR as it has nothing to do with rewriting trace generators, and it is not a small change?